app/javascript/vue/protocol/step.vue

```diff
@@ -440,6 +445,25 @@
       copyPasteImageModal(pasteImages) {
         this.pasteImages = pasteImages;
         this.showClipboardPasteModal = true;
+      },
+      insertElement(element) {
+        let position = element.attributes.position;
+        let elements = this.elements.map( s => {
+          if (s.attributes.position >= position) {
+              s.attributes.position += 1;
+          }
+          return s;
+        })
+        elements.push(element);
+        this.reorderElements(elements);
```

Comment:
maybe we could insert it into the correct position and then we wouldn't need to call reorder